test/Jamfile.v2

```diff
@@ -47,6 +47,7 @@ test-suite variant
       : variant_noexcept_test
     ]
     [ run recursive_variant_test.cpp : : : <rtti>off <define>BOOST_NO_RTTI <define>BOOST_NO_TYPEID : variant_no_rtti_test ]
+    [ run hash_recursive_variant_test.cpp : : : <cxxflags>-std=c++11 : recursive_variant__hash_test ]
```

Comment:
How can I disable non c++11 compilers? It will solve both this and later issue with including unorder_set.